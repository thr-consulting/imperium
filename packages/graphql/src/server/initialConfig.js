export default function initialConfig() {
	return {
		graphql: `${process.env.GRAPHQL_HOST}/api/graphql`,
	};
}
