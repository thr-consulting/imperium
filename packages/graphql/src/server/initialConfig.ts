import {InitialConfig} from '@imperium/core';

export default function initialConfig(): InitialConfig {
	return {
		graphql: `${process.env.GRAPHQL_HOST}/api/graphql`,
	};
}
