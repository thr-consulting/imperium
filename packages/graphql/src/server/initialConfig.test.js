import initialConfig from './initialConfig';

describe('initialConfig', () => {
	it('should return the initial configuration', () => {
		process.env.GRAPHQL_HOST = 'testhost';
		expect(initialConfig()).toMatchSnapshot();
	});
});
