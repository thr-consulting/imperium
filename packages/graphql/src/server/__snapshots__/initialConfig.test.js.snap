// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`initialConfig should return the initial configuration 1`] = `
Object {
  "graphql": "testhost/api/graphql",
}
`;
